routes/raker.router.js

```diff
@@ -0,0 +1,194 @@
+const express = require('express');
+const {db} = require('../server/db')
+const router = express.Router();
+
+//get rakers
+router.get('/:rakers', verifyToken, async (req, res) => {
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'verifyToken' is not defined.